<?xml version="1.0"?>

<launch>
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find rt2_first_assignment)/param/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find rt2_first_assignment)/param/costmap_common_params.yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find rt2_first_assignment)/param/local_costmap_params.yaml" command="load"/>
    <rosparam file="$(find rt2_first_assignment)/param/global_costmap_params.yaml" command="load"/>
    <rosparam file="$(find rt2_first_assignment)/param/base_local_planner_params.yaml" command="load"/>
    <rosparam file="$(find rt2_first_assignment)/param/move_base_params.yaml" command="load"/>
  </node>
</launch>
